src/EKP.jl

```diff
@@ -234,8 +234,8 @@ function update_ensemble!(ek::EKObj{FT, IT, Sampler{FT}}, g) where {FT, IT}
     N_ens = ek.N_ens
 
     # u_mean: N_params x 1
-    u_mean = mean(u', dims=2) 
-    # g_mean: N_params x 1 
+    u_mean = mean(u', dims=2)
+    # g_mean: N_params x 1
     g_mean = mean(g', dims=2)
     # g_cov: N_params x N_params
     g_cov = cov(g, corrected=false)
```

Comment:
corrected=false is a default and can be removed

Reply:
According to the [docs](https://docs.julialang.org/en/v1/stdlib/Statistics/), the default is actually `corrected=true`, so we need to explicitly set the flag to `false` here.